contracts/modules/Checkpoint/ERC20DividendCheckpoint.sol

```diff
@@ -181,8 +181,8 @@ contract ERC20DividendCheckpoint is DividendCheckpoint {
      */
     function reclaimDividend(uint256 _dividendIndex) external withPerm(MANAGE) {
         require(_dividendIndex < dividends.length, "Incorrect dividend index");
-        require(now >= dividends[_dividendIndex].expiry, "Dividend expiry is in the future");
-        require(!dividends[_dividendIndex].reclaimed, "Dividend already claimed");
+        require(now >= dividends[_dividendIndex].expiry, "Invalid dividend expiry.");
+        require(!dividends[_dividendIndex].reclaimed, "Dividend has already been claimed");

```

Comment:
the previous one is ok imo. It's shorter and hence, saves gas. This contract and Security token contract are really on the border of gas limit. Need to keep strings as short as possible on both of these.